wifiphisher/pywifiphisher.py

```diff
@@ -1024,7 +1050,10 @@ def run():
     clients_APs = []
     APs = []
     args.accesspoint = ap_mac
-    args.channel = channel
+
+    if not (args.apname and args.channel):
+        args.channel = channel
```

Comment:
We have a method called `check_args` for these kind of checks. What you want to check here is that both arguments are set and not just the one of them.

Previously, `args.channel` was the channel that we wanted to deauth. You will have to remove all previous occurrences of args.channel since we are not interested in providing this feature anymore.